test/misc_test.rb

```diff
@@ -47,4 +47,83 @@ def test_target_version_outside_developer_env
       end
     end
   end
+
+  def test_target_version_multiple_dbs_below_6
+    skip if multiple_dbs?
+
+    with_target_version({ primary: 10, animals: 15.0 }) do
+      error = assert_raises(RuntimeError) do
+        assert_safe AddColumnDefault
+      end
+      assert_equal "Multiple databases are not supported by this ActiveRecord version", error.message
+    end
+  end
+
+  def test_target_version_multiple_dbs
+    skip unless multiple_dbs?
+
+    with_multiple_dbs do
+
+      safe_version = postgresql? ? 11 : (mysql? ? "8.0.12" : "10.3.2")
+      with_target_version({ primary: safe_version }) do
+        assert_safe AddColumnDefault
+      end
+
+      unsafe_version = postgresql? ? 10 : (mysql? ? "8.0.11" : "10.3.1")
+      with_target_version({ primary: unsafe_version }) do
+        assert_unsafe AddColumnDefault
+      end
+    end
+  end
+
+  def test_target_version_multiple_dbs_unconfigured
+    skip unless multiple_dbs?
+
+    with_multiple_dbs(connects_to: :primary) do
+      error = assert_raises(StandardError) do
+        with_target_version({ animals: 10 }) do
+          assert_safe AddColumnDefault
+        end
+      end
+      assert_equal "StrongMigrations.target_version is not configured for :primary", error.message
+    end
+  end
+
+  private
+
+  def with_multiple_dbs(connects_to: :primary, &block)
```

Comment:
`connects_to` is always `:primary` for the current tests, so no need for a keyword argument.